docs/setup.asciidoc

```diff
@@ -1,33 +1,56 @@
 [[setup]]
-== Getting Kibana Up and Running
+== Installing Kibana
 You can set up Kibana and start exploring your Elasticsearch indices in minutes.
 All you need is:
 
 * Elasticsearch {esversion}
-* A modern web browser - http://www.elastic.co/subscriptions/matrix#matrix_browsers[Supported Browsers].
+* A modern web browser - https://www.elastic.co/support/matrix#show_browsers[Supported Browsers].
 * Information about your Elasticsearch installation:
 ** URL of the Elasticsearch instance you want to connect to.
 ** Which Elasticsearch indices you want to search.
 
 NOTE: If your Elasticsearch installation is protected by http://www.elastic.co/overview/shield/[{scyld}], see
 {shield}/kibana.html#using-kibana4-with-shield[{scyld} with Kibana] for additional setup instructions.
 
+=== Upgrading Kibana
+
+Your existing Kibana version is generally compatible with the next minor version release of Elasticsearch.
+This means you should upgrade your Elasticsearch cluster(s) before or at the same time as Kibana.
+We cannot guarantee compatibility between major version releases so in those cases both Elasticsearch and Kibana must be upgraded together.
+
+. https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html[snapshot] existing `.kibana` index in Elasticsearch
+. save a backup copy of the existing `config/kibana.yml`
+. take note of the Kibana plugins that are installed;
+.. `/opt/kibana/bin/kibana plugin --list` on 4.x versions of Kibana.
+.. `/usr/share/kibana/bin/kibana-plugin --list` on 5.x versions of Kibana.
+. For Kibana installations using an extracted **archive file**;
+.. extract the new version of Kibana into a different directory.  See steps below.
+.. migrate any custom configuration from your old kibana.yml to your new one
+.. follow other steps below to complete the new installation
+.. once the new version is fully configured and working with required plugins, remove the previous version of Kibana
+. For Kibana installations using a Linux **package manager**;
+.. uninstall the existing Kibana package `apt-get remove kibana` or `yum remove kibana`
+.. install the new Kibana package.  See steps below.   (There have been some installer issues between various version of Kibana so the uninstall and install process is safer than an upgrade.)
+
+
 [float]
 [[install]]
-=== Install and Start Kibana
+=== Install and Start Kibana from archive file
```

Comment:
Nit: We generally use "Title Case" for headings: "Install and Start Kibana from an Archive File"

FYI: https://docs.google.com/document/d/1V3YdS8uCA3YNx7cOO_qm92LO9DwuNh7JRDHIsFI9lBQ/